### DISCUSSION TITLE
qiankun主要目和子项目都是umi，主项目entry配成本机ip访问；现在报错我记得需要加个环境的文件来着，忘记是啥了

### DISCUSSION THREAD

**Discussion by QDyanbing** [umijs/umi#11405] [Q&A]
<img width="604" alt="image" src="https://github.com/umijs/umi/assets/66234749/734dd3ee-415a-4867-a046-89a72ac4b2c9">

**Comment by fz6m** [comment]
需要给一个最小复现。

**Comment by QDyanbing** [comment]
已经解决
原需求场景是，本地启动qiankun主项目和qiankun子项目；给其他同事访问；其他同事访问主项目内的路由时正常；通过主项目地址访问子项目路由时报错；
问题发现后，调整主项目加载子项目的entry为本机的局域网ip；访问还是报错；记得当初umi给过方案不过记不清了来着问了下；
经尝试：启动时附加环境变量HOST=本机ip可解决；（当初umi给的方案应该是.env文件中加入HOST=本机ip）昨天尝试的时候可能是没重启的原因一直没生效；
